### DISCUSSION TITLE
大佬，封禁后还能申请一次解封吗

### DISCUSSION THREAD

**Discussion by Xand233** [BurnHair/docs#148] [技术支持]
推黄油里忘了关gpt了，本来以为收费用户不限内容，后来才看到有限制内容，大佬能给次机会吗😂

**Comment by BennyThink** [comment]
仅此一次哦 已经可以了。

**Reply by Xand233** [reply]
> 仅此一次哦 已经可以了。

感谢，以后谨慎点了，有时候看个油管视频也能蹦出来些限制词😂
